src/coredump/coredumpctl.c

```diff
@@ -790,9 +791,10 @@ static int print_info(FILE *file, sd_journal *j, bool need_space) {
                 _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
 
                 r = json_parse(pkgmeta_json, 0, &v, NULL, NULL);
-                if (r < 0)
-                        log_warning_errno(r, "json_parse on %s failed, ignoring: %m", pkgmeta_json);
-                else {
+                if (r < 0) {
+                        _cleanup_free_ char *esc = cescape(pkgmeta_json);
+                        log_warning_errno(r, "json_parse on %s failed, ignoring: %m", strnull(esc));
```

Comment:
given that the string that is output here is possibly total garbage maybe enclose in the output in quotes?, i.e. `%s` → `'%s'`

or so?

Reply:
Updated. I used double quotes though, because I think they look better with c-escaped strings.